move_base_msgs/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ add_message_files(
   FILES
     Feedback.msg
     RecoveryStatus.msg
+    Stuck.msg
```

Comment:
What's this?

Reply:
The robot stack message.

Reply:
What's the meaning and motivation for this message? We can't add messages if we ever want this to be contributed.

Reply:
But, OMS should know if robot is stuck or not.I have move_base PR
[https://github.com/Blindnology/navigation/pull/4](https://github.com/Blindnology/navigation/pull/4)]